features/login/impl/src/main/kotlin/io/element/android/features/login/impl/screens/loginpassword/LoginPasswordView.kt

```diff
@@ -272,6 +275,13 @@ private fun LoginForm(
     }
 }
 
+/**
+ * Ensure that the string does not contain any new line characters, which can happen when pasting values.
+ */
+private fun String.sanitize(): String {
+    return replace("\n", "")
```

Comment:
I'm checking with some other apps and most of them translate `\n` into a single whitespace. Maybe we should do the same, but I don't have a strong opinion on this.

Reply:
Yes, I was wondering as well. I think we can keep it like that, it's quite rare to have this user story anyway.